src/pages/pass-finalisation/cat-c/_tests_/pass-finalisation.cat-be.page.spec.ts

```diff
@@ -0,0 +1,180 @@
+import { AuthenticationProvider } from '../../../../providers/authentication/authentication';
+import { AuthenticationProviderMock } from '../../../../providers/authentication/__mocks__/authentication.mock';
+import { DateTimeProvider } from '../../../../providers/date-time/date-time';
+import { DateTimeProviderMock } from '../../../../providers/date-time/__mocks__/date-time.mock';
+import { Store } from '@ngrx/store';
+import { StoreModel } from '../../../../shared/models/store.model';
+import { PersistTests } from '../../../../modules/tests/tests.actions';
+import { MockComponent } from 'ng-mocks';
+import { PracticeModeBanner } from '../../../../components/common/practice-mode-banner/practice-mode-banner';
+import { PassCertificateNumberComponent } from '../../components/pass-certificate-number/pass-certificate-number';
+import { LicenseProvidedComponent } from '../../components/license-provided/license-provided';
+import { D255Component } from '../../../../components/test-finalisation/d255/d255';
+import { DebriefWitnessedComponent } from
+  '../../../../components/test-finalisation/debrief-witnessed/debrief-witnessed';
+import { LanguagePreferencesComponent } from
+  '../../../../components/test-finalisation/language-preference/language-preferences';
+import { FinalisationHeaderComponent } from
+  '../../../../components/test-finalisation/finalisation-header/finalisation-header';
+import { ProvisionalLicenseReceived, ProvisionalLicenseNotReceived, PassCertificateNumberChanged } from
+  '../../../../modules/tests/pass-completion/pass-completion.actions';
+import {
+  PassFinalisationViewDidEnter,
+  PassFinalisationValidationError,
+} from '../../pass-finalisation.actions';
+import { GearboxCategoryChanged } from '../../../../modules/tests/vehicle-details/vehicle-details.actions';
+import { D255Yes, D255No, DebriefWitnessed, DebriefUnwitnessed } from
+  '../../../../modules/tests/test-summary/test-summary.actions';
+import { CandidateChoseToProceedWithTestInWelsh, CandidateChoseToProceedWithTestInEnglish } from
+  '../../../../modules/tests/communication-preferences/communication-preferences.actions';
+import { PassFinalisationCatCPage } from '../pass-finalisation.cat-c.page';
+import { WarningBannerComponent } from '../../../../components/common/warning-banner/warning-banner';
+import { TransmissionComponent } from '../../../../components/common/transmission/transmission';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+import { PASS_CERTIFICATE_NUMBER_CTRL }
+  from '../../components/pass-certificate-number/pass-certificate-number.constants';
+
+describe('PassFinalisationCatCPage', () => {
+  let fixture: ComponentFixture<PassFinalisationCatCPage>;
+  let component: PassFinalisationCatCPage;
+  let store$: Store<StoreModel>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        PassFinalisationCatCPage,
+        MockComponent(PracticeModeBanner),
```

Comment:
Can remove component and import.

Reply:
removed, and also change the name of the file